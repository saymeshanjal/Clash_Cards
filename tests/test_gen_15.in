21
14 177 2
14 247 2
6 16 5
6 19 3
2 206 2
16 229 2
12 156 1
16 5 2
6 48 3
4 63 1
12 116 4
16 6 2
2 8 2
6 130 4
16 47 4
8 21 5
2 96 3
6 227 2
12 23 3
10 46 5
12 122 1