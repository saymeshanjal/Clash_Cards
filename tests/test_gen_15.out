1078 1079 1275
1078 1079 1275
1 19 0
6 14 0
0 1 1
4081 4082 4707
462 462 0
495 4257 8118
6 14 0
3 3 0
208 441 275
165 4257 8448
0 1 1
1 14 5
373 8838 3659
1 69 0
0 2 0
5 6 9
146 778 0
45 207 0
462 462 0
